operations/messagethreads.md

```diff
@@ -0,0 +1,109 @@
+# Message threads
+
+## Get all message threads
+
+Returns all message threads created by the integration.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/messageThreads/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "CreatedUtc":{
+        "StartUtc": "2022-03-03T00:00:00Z",
+        "EndUtc": "2022-03-14T00:00:00Z"
+    }
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `CreatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Message thread](#message-thread) was created. |
+| `UpdatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Message thread](#message-thread) was updated. |

```

Comment:
Missing `MessageThreadIds`. Can use something like `optional, required if no other filter is provided` 

Also newline